README.md

```diff
@@ -6,23 +6,23 @@ Generalization of [Chaumian CoinJoins](https://github.com/nopara73/ZeroLink/). E
 
 ## Background
 
-A Bitcoin transaction consists of inputs and outputs. [CoinJoins](https://bitcointalk.org/index.php?topic=279249.msg2983902#msg2983902) are Bitcoin transactions of multiple collaborating participants. This construction is inherently secure, because if a participant does not see its desired outputs in the resulting CoinJoin, it refuses to sign its provided inputs.
+A Bitcoin transaction consists of inputs and outputs. CoinJoins[REFERENCE] are Bitcoin transactions where multiple participants collaborate on. This construction is inherently secure, because if a participant does not see its desired outputs in the resulting CoinJoin, it refuses to sign its provided inputs.
```

Comment:
```suggestion
A Bitcoin transaction consists of inputs and outputs. [CoinJoins](https://bitcointalk.org/index.php?topic=279249.msg2983902#msg2983902) are Bitcoin transactions of multiple collaborating participants. This construction is inherently secure, because if a participant does not see its desired outputs in the resulting CoinJoin, it refuses to sign its provided inputs.
```

Oh, my other PR got merged while I was editing this branch - fixing a conflict here.